app/helpers/application_helper.rb

```diff
@@ -21,6 +21,14 @@ def organization_abbreviation_dropdown_options
     Organization.all.order(:name).map { |org| ["#{org.abbreviation} - #{org.name}", org.abbreviation] }
   end
 
+  def us_timezones
```

Comment:
only show these timezones, rather than everything from around the world

though USAID may want/need other times